css/style.css

```diff
@@ -0,0 +1,247 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    background-color: rgb(238, 238, 238);
+}
+
+header{
+    background-color: rgb(238, 238, 238);
+    padding: 10px;
```

Comment:
I think it would be better to use more responsive measurements such as rem, em, and % instead of px